modules/ROOT/pages/test-recorder.adoc

```diff
@@ -0,0 +1,49 @@
+= Test Recorder in Studio (Beta)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The test recorder enables you to record a processing flow and then configure a unit test based on the captured event.
+
+The test recorder captures real data as it goes through your application in Studio. By capturing the flow execution, MUnit can automatically generate an MUnit test so that you can configure the necessary mocks and assertions for your test. See xref:test-recorder-examples.adoc[Recording a Test from an Exchange Example] for a fully working example.
+
+[NOTE]
+The test recorder is an open beta feature.
+
+== Compatibility
+
+The following table describes the required product versions to use the test recorder. These versions are for the beta feature and are likely to change for the GA (general availability) release.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Product | Version
+| Anypoint Studio
+| 7.4.0 and later.
+
+| MUnit
+| 2.2.4
+
+| MUnit Anypoint Studio Plugin
+| 2.4.2
+
+| Mule runtime engine
+| 4.2.2
+
+|===
+
+== Limitations
+
+The test recorder has the following limitations for creating MUnit tests:
+
+* You cannot create tests for flows that have mule errors either raised inside the flow or already inside the incoming event. Not even if they are handled by an *on error continue* error handler.
+* When the recorded flow execution ends, the application is killed. This causes the result of the execution to never reach its destination. For example, an HTTP listener will not respond to the incoming request. This does not mean that the recording failed.
```

Comment:
Limit list items and steps to one sentence maximum, to enhance scannability (which is the purpose of a list).
Also, remember that the items in this list are supposed to be limitations, so begin each with what the real limitation is: in this case, I think the limitation is that the result of the flow does not reach its destination?
This is just my guess; please reword for accuracy, as appropriate:
```suggestion
* A recorded flow execution ends successfully, but the result does not reach its destination because the application is killed. 
```